sklearn/metrics/_classification.py

```diff
@@ -1348,6 +1348,12 @@ def fbeta_score(
     averaged depending on the `average` parameter. Use `labels` specify the set of
     labels to calculate F-beta score for.
 
+    F-beta score is not implemented as a named scorer that can be
+    passed to the `scoring` parameter directly; :func:`make_scorer`
+    needs to be used first instead to create a callable object to then
+    pass to the `scoring` parameter, e.g. like: ``fbeta_scorer =
+    make_scorer(fbeta_score, beta=2)``
```

Comment:
This last example could be used in the "Examples" section actually by showing how to pass the scorer to an estimator exposing a `scoring` parameter.

In this way, the note is only informing about not exposing a string-like scorer.

Reply:
I'll move my addition to the Notes section and add the longer Example then as well